android/jni/com/mapswithme/core/jni_helper.cpp

```diff
@@ -109,6 +109,15 @@ namespace jni
     return env->NewStringUTF(s);
   }
 
+  jobjectArray ToJavaStringArray(JNIEnv * env, vector<string> const & src)
```

Comment:
Нужно, но делать это уже вызывающий код должен, иначе он получит невалидную локальную ссылку.

> On Apr 28, 2016, at 12:16 PM, Alex Zolotarev <notifications@github.com> wrote:
> 
> In android/jni/com/mapswithme/core/jni_helper.cpp <https://github.com/mapsme/omim/pull/3042#discussion_r61396593>:
> 
> > @@ -109,6 +109,15 @@ namespace jni
> >      return env->NewStringUTF(s);
> >    }
> >  
> > +  jobjectArray ToJavaStringArray(JNIEnv * env, vector<string> const & src)
> Это не нужно потом освобождать? Может в умный указатель лучше класть?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3042/files/4ec789b43f920686cf11edafc48c5e58449ba38a#r61396593>